subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy

```diff
@@ -129,6 +129,13 @@ class BuildScanPluginSmokeTest extends AbstractSmokeTest {
 
         then:
         scanRunner()
+            .expectLegacyDeprecationWarningIf(versionNumber < FIRST_VERSION_CALLING_BUILD_PATH,
+                "Usage of the Gradle Enterprise plugin " + version + " has been deprecated. " +
+                    "This will change in Gradle 9.0. " +
```

Comment:
I find the "This will change in Gradle 9.0" strange. It sounds like the deprecation will change in Gradle 9. Though I don't have a better suggestion that fits with our usual deprecation options. Maybe we can change the first sentence "Usage of ..." a little bit so this makes more sense? @lkasso Do you have some ideas on how to make the whole deprecation message better?

Reply:
@alllex @wolfs,
Right now, it's confusing because:

- It says `3.13` is deprecated, but `3.13.1` should be used. A user like my is thinking, isn't `3.13.1` part of `3.13`? I am not sure what the convention is, though.
- "This will change in Gradle 9.0" is not a complete sentence because it's not actionable for the user

Also, we agreed on no use of the word "Please".

What about something like:

**Gradle Enterprise plugin 3.13.0 has been deprecated.** 
Starting with Gradle 9.0, only Gradle Enterprise plugin 3.13.1 (or newer) is supported.

_OR (depending on convention)_

**Gradle Enterprise plugin 3.13 has been deprecated.** 
Starting with Gradle 9.0, only Gradle Enterprise plugin 3.13.1+ is supported.